lib/cli.py

```diff
@@ -22,10 +22,12 @@ def __call__(self, parser, namespace, values, option_string=None):
 class TrainingProcessor(object):
     arguments = None
 
-    def __init__(self, description='default'):
+    def __init__(self, subparser, command, description='default'):
         print('Initializing')
```

Comment:
All three actions/scripts/objects are initialized when `faceswap.py` is run. Maybe alter or remove these print statements, or only initialize only object that belongs to the subcommand we are actually running.

```
> python .\faceswap.py
Using TensorFlow backend.
Initializing
Initializing
Initializing
```